include/astshim/Channel.h

```diff
@@ -76,8 +76,10 @@ class Channel : public Object {
     virtual ~Channel();
 
     Channel(Channel const &) = delete;
+    /// Move constructor
```

Comment:
The Stack-wide Doxygen config in `base` (which `astshim` doesn't depend on) sets `EXTRACT_ALL = YES`, which disables warnings about undocumented members as a side effect. I'd propose depending on `base` instead of manually adding no-op documentation.